src/main/java/frc/robot/subsystems/ShooterAdjuster.java

```diff
@@ -0,0 +1,22 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.CANSparkMax;
+import com.spikes2212.command.genericsubsystem.smartmotorcontrollersubsystem.SparkGenericSubsystem;
+import edu.wpi.first.wpilibj.DutyCycleEncoder;
+
+public class ShooterAdjuster extends SparkGenericSubsystem {
```

Comment:
there are limit switches at the top and the bottom, make sure to add them to the code